kata-deploy/scripts/install-kata-crio.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+echo "copying kata artifacts from /tmp to /opt"
+cp -R /tmp/kata/* /opt/kata/
+
+chmod +x /opt/kata/bin/*
+
+cp /opt/kata/configuration.toml /usr/share/defaults/kata-containers/configuration.toml
+
+cp /etc/crio/crio.conf /etc/crio/crio.conf.bak
+
+echo "Set Kata containers as default runtime in CRI-O for untrusted workloads"
+ sed -i 's/runtime_untrusted_workload = ""/runtime_untrusted_workload = "\/opt\/kata\/bin\/kata-runtime"/' /etc/crio/crio.conf
```

Comment:
remove space in the beginning, for sure change the untrusted setting to kata and no need for escape characters - 
```
sed -i '/runtime_untrusted_workload = /c\runtime_untrusted_workload = "/opt/kata/bin/kata-runtime"' /etc/crio/crio.conf
```

Reply:
@jodh-intel, @chavafg - I did this based on our CI scripts -- can you comment?

Reply:
I agree with @krsna1729 as the code is easier to read without requireing backslashes. However, I think the command would need to be something like the following with uses `!` as the delimiter for `sed`:

```
sed -i 's!runtime_untrusted_workload = ""!runtime_untrusted_workload = "/opt/kata/bin/kata-runtime"!' /etc/crio/crio.conf
```

Reply:
So if there is a `runtime_untrusted_workload` already registered, @jodh-intel's version nor the original one would change that. The node would still be labelled `kata-runtime=true` (this part is in the yaml)? Should we pull that in here or exit with error from here so that the said label will not be applied.

Reply:
Ah, that's a good question.  Since they are explicitly trying to install Kata, I think the best thing may be to just overwrite what was there before.  WDYT @krsna1729

Reply:
I'll take your input, @krsna1729, so I just do a line replace instead of matching in ""